### ISSUE TITLE
Add prebuilt-installation option

### ISSUE THREAD

**Issue by rkhamis** [threefoldtech/jumpscaleX_core#88]
This installation option will allow quick-start to a ready-built threebot.

Requires: threefoldtech/jumpscaleX_builders#27

specs:
add installation option `--prebuilt` which will clone https://github.com/threefoldtech/threebot_prebuilt and the jumpscale repos, allowing for an out of the box development ready state.

**Comment by alichaddad** [comment]
Prebuilt sandbox is ready and already pushed, and the `prebuilt` feature was added to the installer.
The following still remains to be done:

TODO
- change image 3bot to remove all links under `/sandbox/lib/jumpscale`
- Test that threebotserver,etc,.. starts properly

Branches
Builder: development_threebot_sandbox
Core: development_threebot_sandbox

**Comment by BolaNasr** [comment]
change image 3bot to https://hub.docker.com/r/3bot/3bot 
TODO : 

- Test that threebotserver,etc,.. starts properly

**Comment by rkhamis** [comment]
no longer relevant
